### DISCUSSION TITLE
force goreleaser to pick correct tag

### DISCUSSION THREAD

**Discussion by theganyo** [goreleaser/goreleaser#1965] [Q&A] [answered]
I'm running into an issue where if a specific commit is tagged more than once, I can't see a way to force goreleaser to use the latest tag for the release - instead, it re-releases on the first tag.

It may be esoteric, but there are a couple of ways I've had this happen:

* I keep multiple repos in sync version-wise, so there are times when I tag a new release without changing the commit.
* I want to tag a release candidate commit as a final release.

It would be great if there's a way to address this automatically (ie. get the most recent tag), but if not, perhaps there could be a way to tell goreleaser to use a specific tag?

**Comment by caarlos0** [comment, marked as answer]
you can set GORELEASER_CURRENT_TAG: https://goreleaser.com/customization/build#define-build-tag

doesn't that work?

**Comment by theganyo** [comment]
Ah! Thank you. I hadn't seen that. I'll give it a try. Closing this.
